jni/BUILD.adoc

```diff
@@ -89,6 +98,13 @@ The instructions below assume that you start each build in `/path/to/kotlin-satl
  install -m 644 src/cadical.hpp -Dt install/include/cadical
  install -m 644 build/libcadical.so -Dt install/lib
 
+* 👨🏻‍💻 Build and install on macOS:
+
+ ./configure -fPIC
+ make -j16 dynamiclib
+ mkdir -p install/include/cadical && cp src/cadical.hpp install/include/cadical
+ mkdir install/lib && cp build/libcadical.dylib install/lib
```

Comment:
Why not just use `install`? OSX does not have `install`? 🤔

Reply:
OSX has `install`, but it doesn't have the same behavior